packages/ethereum-storage/src/ethereum-storage.ts

```diff
@@ -464,6 +464,128 @@ export default class EthereumStorage implements StorageTypes.IStorage {
     };
   }
 
+  /**
+   * Verify the hashes are present on IPFS for the corresponding ethereum entry
+   * Filtered incorrect hashes
+   * public to facilitate the tests
```

Comment:
If you make a method public to facilitate tests, I think it's not at the right place... maybe should be a helper outside this class, or moved to its own class

Reply:
I moved it in another file :) hope it is better !